generators/firechip/src/main/scala/FireSim.scala

```diff
@@ -38,44 +40,113 @@ object NodeIdx {
 /**
   * Under FireSim's current multiclock implementation there can be only a
   * single clock bridge. This requires, therefore, that it  be instantiated in
-  * the harness and reused across all supernode instances. This class attempts to 
+  * the harness and reused across all supernode instances. This class attempts to
   * memoize its instantiation such that it can be referenced from within a ClockScheme function.
   */
 class ClockBridgeInstantiator {
-  private var _clockRecord: Option[RecordMap[Clock]] = None
+  private var _harnessClockMap: HashMap[String, (Double, Clock)] = HashMap.empty
```

Comment:
I think you iterate over these two hashmaps later but I can't tell if that will affect the verilog naming or not.

Reply:
I never iterate over the hashmaps to determine Verilog signals. I only use it to "store" requests (clock wires) and later connect them to the clock bridge.

Reply:
Seems like most people care about ordered maps so Ill change this.